gridkit/base_grid.py

```diff
@@ -420,9 +421,9 @@ def interp_from_points(
             max(coords[1]),
         )
         aligned_bounds = self.align_bounds(bounds, mode="expand")
-        ids, shape = self.cells_in_bounds(aligned_bounds)
+        ids = self.cells_in_bounds(aligned_bounds)
         interp_values = numpy.full(
-            shape=shape, fill_value=nodata_value, dtype=values.dtype
+            shape=ids.shape, fill_value=nodata_value, dtype=values.dtype

```

Comment:
Here too, make sure this works in both the case where cells_in_bounds returns a 1D and a 2D index.
Use `return_cell_count=True` if it does not work for both cases.